app/decorators/rejection/reason_decorator.rb

```diff
@@ -0,0 +1,17 @@
+class Rejection::ReasonDecorator < Draper::Decorator
+  delegate_all
+  def checkbox_for(reason, html_options = {})
+    html_options[:id] = nil unless html_options.key?(:id)
```

Comment:
There is quite a lot going in this method, how about splitting some of the logic into private methods.

One could be to setup the default values on `html_options` (making sure it has keys for `id`, `data` and `class` and then another method that returns the `class` value ie html_class